src/main/scala/com/gossiperl/client/actors/ActorRegistry.scala

```diff
@@ -0,0 +1,60 @@
+package com.gossiperl.client.actors
+
+import akka.actor.{ActorRef, Actor}
+import com.typesafe.scalalogging.slf4j.LazyLogging
+
+import scala.concurrent.{Future, Promise}
+
+class ActorNotRegisteredException(message: String) extends RuntimeException(message)
+
+object ActorRegistry extends LazyLogging {
```

Comment:
I was not aware of the built in class. I think this is a registry of `Props(Class)`, not `running actors` as such.